### ISSUE TITLE
Set region return wrong message

### ISSUE THREAD

**Issue by SakaYK** [civo/cli#98]
using civo cli version 0.7.22 in linux, return wrong message when success set the region.

```
❯ civo version
Civo CLI v0.7.22

❯ civo region set LON1
The default region was set to (London 1) LON1
The region you tried to set LON1 doesn't exist, please use 'civo region ls' to get the code of a valid region

❯ civo region ls
+------+-------------+----------------+---------+
| Code | Name        | Country        | Current |
+------+-------------+----------------+---------+
| LON1 | London 1    | United Kingdom | <=====  |
| NYC1 | New York 1  | United States  |         |
| SVG1 | Stevenage 1 | United Kingdom |         |
+------+-------------+----------------+---------+

❯ civo region set NYC1
The default region was set to (New York 1) NYC1
The region you tried to set NYC1 doesn't exist, please use 'civo region ls' to get the code of a valid region

❯ civo region ls
+------+-------------+----------------+---------+
| Code | Name        | Country        | Current |
+------+-------------+----------------+---------+
| LON1 | London 1    | United Kingdom |         |
| NYC1 | New York 1  | United States  | <=====  |
| SVG1 | Stevenage 1 | United Kingdom |         |
+------+-------------+----------------+---------+
```

**Comment by alejandrojnm** [comment]
Hi @SakaYK i will check this, thank for the repot

**Comment by alejandrojnm** [comment]
This is already fixed in the v0.7.23
